libtransmission/rpc-server.cc

```diff
@@ -882,201 +880,53 @@ void tr_rpc_server::setAntiBruteForceEnabled(bool enabled) noexcept
 *****  LIFE CYCLE
 ****/
 
-static void missing_settings_key(tr_quark const q)
-{
-    tr_logAddDebug(fmt::format("Couldn't find settings key '{}'", tr_quark_get_string_view(q)));
-}
-
 tr_rpc_server::tr_rpc_server(tr_session* session_in, tr_variant* settings)
     : compressor{ libdeflate_alloc_compressor(DeflateLevel), libdeflate_free_compressor }
     , web_client_dir_{ tr_getWebClientDir(session_in) }
-    , bindAddress(std::make_unique<struct tr_rpc_address>())
+    , bind_address_(std::make_unique<struct tr_rpc_address>()) // TODO(ckerr) why is thnis a unique_ptr
```

Comment:
typo thnis -> this

Reply:
Fixed in b83efa0a597f3cf39ca4921a1190da6e7c61d1a0

(Answer to the "why" question: because the implementation is private inside rpc-server.cc)